standalone-metastore/metastore-server/src/main/sql/derby/hive-schema-4.1.0.derby.sql

```diff
@@ -269,7 +269,7 @@ CREATE UNIQUE INDEX "APP"."ROLEENTITYINDEX" ON "APP"."ROLES" ("ROLE_NAME");
 
 CREATE INDEX "APP"."TABLEPRIVILEGEINDEX" ON "APP"."TBL_PRIVS" ("AUTHORIZER", "TBL_ID", "PRINCIPAL_NAME", "PRINCIPAL_TYPE", "TBL_PRIV", "GRANTOR", "GRANTOR_TYPE");
 
-CREATE UNIQUE INDEX "APP"."UNIQUETABLE" ON "APP"."TBLS" ("TBL_NAME", "DB_ID");
+CREATE UNIQUE INDEX "APP"."UNIQUETABLE" ON "APP"."TBLS" ("DB_ID", "TBL_NAME");
```

Comment:
what does it change?

Reply:
We change the order of this composite index to improve performance. Because we always get the`DB_ID` first while doing any query for hive table, so the first level (DB_ID) of the index alway take effect after this change.

It's a similar change like https://github.com/apache/hive/pull/4831#pullrequestreview-1814098075

Reply:
are you saying we won't hit the index if the order of columns is reversed?

Reply:
It could only hit the first part (`TBL_NAME`) of `UNIQUETABLE` index currently if we query tables with table pattern because it will use `like clause` for `TBL_NAME`. If we switch the order, both `DB_ID` and `TBL_NAME` can be hit.